MeNext.Spotify.iOS/SpotifyAuth.cs

```diff
@@ -82,6 +85,11 @@ public StreamingDelegate CreateStreamingDelegate()
             return sd;
         }
 
+        /// <summary>
+        /// Opens the URL.
```

Comment:
Wrong. This is called when the app is opened via a URL (namely `me-next://callback`)